osgp/platform/osgp-adapter-domain-smartmetering/src/main/java/org/opensmartgridplatform/adapter/domain/smartmetering/infra/jms/ws/WebServiceResponseMessageSender.java

```diff
@@ -41,13 +42,13 @@
     public void send(final ResponseMessage responseMessage, final Long timeToLive, final String messageType) {
 
         // Keep the original time to live from configuration.
-        final Long originalTimeToLive = this.webServiceResponsesJmsTemplate.getTimeToLive();
+        final Long originalTimeToLive = this.jmsTemplate.getTimeToLive();
         if (timeToLive != null) {
             // Set the custom time to live.
-            this.webServiceResponsesJmsTemplate.setTimeToLive(timeToLive);
+            this.jmsTemplate.setTimeToLive(timeToLive);
         }
 
-        this.webServiceResponsesJmsTemplate.send(new MessageCreator() {
+        this.jmsTemplate.send(new MessageCreator() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce this anonymous class number of lines from 28 to at most 20, or make it a named class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1188)